bigquery/tests/system.py

```diff
@@ -775,6 +778,13 @@ def test_sync_query_w_query_params(self):
             self.assertEqual(len(query.rows), 1)
             self.assertEqual(len(query.rows[0]), 1)
             self.assertEqual(query.rows[0][0], example['expected'])
+            if 'query_parameters' not in example:
+                Config.CURSOR.execute(example['sql'])
+                row = Config.CURSOR.fetchone()
+                self.assertEqual(len(row), 1)
+                self.assertEqual(row[0], example['expected'])
+                row = Config.CURSOR.fetchone()
+                self.assertIsNone(row)
```

Comment:
Needs to have the `dbapi` tests separated out into their own methods (factoring out the generation of `EXAMPLES`.